cmd/immortal/main.go

```diff
@@ -49,11 +48,22 @@ func main() {
 	if len(cfg.Require) > 0 {
 		down := []string{}
 		ctl := &immortal.Controller{}
+		services, _ := ctl.FindServices(immortal.GetSdir())
```

Comment:
This will load all the existing services not only the ones you may just want to check for